### PR TITLE
[Nautilus]: Add missing tier-2 testsuites to metadata file

### PR DESCRIPTION
**Pull request by ckulal**
Signed-off-by: ckulal <ckulal@redhat.com>
[nautilus]: Add missing tier-2 test suites(which was previously part of scripts->tier-3 place holder in pipeline v2)
from RGW FG to 4.3 meta data file.

logs:
tier-2_rgw_multisite_test-bucket-lifecycle-primary-to-secondary(1hr 25min)
https://159.23.92.24/job/rhceph-test-execution-manual/48/console

tier-2_rgw_multisite_test-bucket-lifecycle-secondary-to-primary(95 min)
http://magna002.ceph.redhat.com/ceph-qe-logs/Chaithra/cephci-run-JU3873/

tier-2_rgw_test-multisite-metadata-sync(1hr 12 min)
https://159.23.92.24/job/rhceph-test-execution-manual/40/console

tier-2_rgw_image_test-bucket-lifecycle-and-listing(89 min)
http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-OVQ3LZ/

tier-2_rgw_s3cmd_test-bucket-object-stats(24 min)
http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-20U4GV/

tier-2_rgw_s3tests(45 min)
http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-86GIFI/

tier-2_rgw_test-bucket-acls-and-links(26 min)
http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-SVMNB2/

tier-2_rgw_test-bucket-notifications(32 min)
http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-ZGBXJP/

tier-2_rgw_test-bucket-ops-and-versioning(29 min)
http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-6HWD0H/

2) removed verify-io from LC Mult-isite test suite. Will be added back once the verify-io is enhanced to support delete operation verification.

Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs
<details>

<summary>click to expand checklist</summary>

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarion Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test
</details>

### CODE REVIEW

**Review by anrao19 — APPROVED** [review]
LGTM

File: pipeline/metadata/4.3.yaml [hunk 1]
```diff
@@ -589,3 +589,147 @@
...
+  inventory:
+    openstack: "conf/inventory/rhel-8-latest.yaml"
+    ibmc: "conf/inventory/ibm-vpc-rhel-8-latest.yaml"
+  metadata:
+    - schedule
+    - tier-2
+    - openstack
+    - ibmc
+    - rgw
+    - stage-2
```
Comment (psathyan) [inline pipeline/metadata/4.3.yaml:607]: We already have 5 test suites in `tier-2 & stage-2`... could we look at the next stages?

Comment (ckulal) [reply pipeline/metadata/4.3.yaml:607]: For a scheduler job there is only one suite under tier-2 stage-2,
so included these suites under same stage.

Comment (psathyan) [reply pipeline/metadata/4.3.yaml:607]: Thanks for clarifying... I missed the `schedule` tag.

**Pull request merged**